common/models/model-definition.json

```diff
@@ -96,7 +96,8 @@
     "methods": {
       "embed": {
         "as": "object",
-        "key": "name"
+        "keySetter": "name",
```

Comment:
Since the conversion from the key in JSON file into name/isStatic in data is already handled by `getDataFromConfig`, `keySetter` is no longer needed and can be removed. Unless I am missing something?